lib/prelude.mli

```diff
@@ -0,0 +1 @@
+val env_and_file : string -> string option * string
```

Comment:
what does this function do?

Reply:
This function was moved from the `bin/main/test.ml` where it was used to parse passed through `--prelude` or `--prelude-str` to separate the filename/content from the env into which the prelude must be evaluated.

That's where the `Prelude` name comes from. I extracted it here because it is now also used in `bin/rule.ml`. If you have a better suited module in mind, I'll happily move it there.